### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.iot.device.common.transport_exceptions.UnauthorizedError
  fullName: azure.iot.device.common.transport_exceptions.UnauthorizedError
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - type: builtins.object
        type: builtins.BaseException
      type: builtins.Exception
    type: azure.iot.device.common.chainable_exception.ChainableException
  langs:
  - python
  module: azure.iot.device.common.transport_exceptions
  name: UnauthorizedError
  source:
    id: UnauthorizedError
    path: azure-iot-device\azure\iot\device\common\transport_exceptions.py
    remote:
      branch: master
      path: azure-iot-device\azure\iot\device\common\transport_exceptions.py
      repo: https://github.com/Azure/azure-iot-sdk-python
    startLine: 27
  summary: Authorization was rejected
  syntax:
    content: UnauthorizedError(message=None, cause=None)
  type: class
  uid: azure.iot.device.common.transport_exceptions.UnauthorizedError
references: []
